Lorem Ipsum XR Translator/Assets/Scripts/ObjectDetection.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.UI;
+using TMPro;
+
+public class ObjectDetection : MonoBehaviour
+{
+    public TextMeshPro objCategory;
+    public Image image;
+    private readonly string subscriptionKey = "c2242d7717124c79baa26bd78f027d8b";
```

Comment:
I'd recommend revoking this key now that it's public - consider storing this separately to the code

Reply:
@brwhelan-msft , thank you. The key does not work anymore. Already generated a new one which is in use for testing now.